src/app/state/staking/saga.ts

```diff
@@ -82,16 +85,113 @@ function* loadDebondingDelegations(publicKey: Uint8Array) {
   return debondingDelegations
 }
 
-function* refreshValidators() {
+export function* maybeRefreshValidators() {
```

Comment:
Prefer a previous name without maybe, this one make me think that I have no idea what to expect.

Reply:
Alright :) Didn't know how to name it, after I added
```ts
  // Skip if validators are already in store, and network didn't change
  if (existingValidators.length > 0 && network === existingValidatorsNetwork) return
```